Q7/harrypahwa97/SieveOfEratosthenes.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+public class SieveOfEratosthenes {
+	public static void main(String harsh[]){
+		Scanner s=new Scanner(System.in);
+		int N = s.nextInt();
+		int PrimeArray[]=new int[1000001];
```

Comment:
Why this huge sized array? @harrypahwa97

Reply:
So Yatharth, NO it can't be imporved against size as generally in most of the competitive coding platforms they have a test case for N<=10^5 and 10^7 as well so this code works well in that case as well.
for eg find the prime numbers till 1000000